runtime/runtime/src/lib.rs

```diff
@@ -1120,6 +1125,23 @@ impl Runtime {
                 &mut stats,
             )?;
         }
+        if !apply_state.is_new_chunk
+            && apply_state.current_protocol_version
+                >= PROTOCOL_FEATURES_TO_VERSION_MAPPING[&ProtocolFeature::FixApplyChunks]
+        {
```

Comment:
Should we use `checked_feature!`?

Reply:
checked_feature does not work when there is no associated rust feature